src/neo-vm/ExecutionEngineLimits.cs

```diff
@@ -48,6 +48,11 @@ public sealed record ExecutionEngineLimits
         /// </summary>
         public uint MaxTryNestingDepth { get; init; } = 16;
 
+        /// <summary>
+        /// Allow to catch the ExecutionEngine Exceptions
+        /// </summary>
+        public bool CatchEngineExceptions { get; init; } = true;
```

Comment:
@erikzhang We can set this var to `false` before X block

Reply:
I am not sure if it is necessary.